frontend/src/mocks/msw/handlers/boards/boardsResponse.js

```diff
@@ -1,3 +1,5 @@
+import avatar1 from 'mocks/images/avatars/sample-avatar1.jpg';
+import avatar2 from 'mocks/images/avatars/sample-avatar2.jpg';
```

Comment:
Ja tutaj u siebie wrzuciłem ikonki zamiast avatara. 
Dotyczy to bardziej boardu niż użytkownika, więc myślę, że ikonki bardziej wskazane.